static/src/javascripts/projects/commercial/modules/dfp/render-advert.ts

```diff
@@ -18,9 +18,17 @@ import { renderAdvertLabel } from './render-advert-label';
  *
  */
 
+type Callback = {
```

Comment:
Great work! Minor but could we bit a bit more specific with the naming of `callback` which tells more about what the function is where it's used?

Reply:
Good shout!